src/routes/docs/products/databases/atomic-numeric-operations/+page.markdoc

```diff
@@ -0,0 +1,524 @@
+client.set_key('<API_KEY>') # Your secret API key
+
+databases = Databases(client)
+
+result = databases.increment_document_attribute(
+    database_id = '<DATABASE_ID>',
+    collection_id = '<COLLECTION_ID>',
+    document_id = '<DOCUMENT_ID>',
+    attribute = 'likes', # attribute
+    value = 1 # value
+)
+```
+```graphql
+mutation {
+    databasesIncrementDocumentAttribute(
+        databaseId: "<DATABASE_ID>",
+        collectionId: "<COLLECTION_ID>",
+        documentId: "<DOCUMENT_ID>",
+        attribute: "likes",
+        value: 1
+    ) {
+        _id
+        _collectionId
+        _databaseId
+        _createdAt
+        _updatedAt
+        _permissions
+        data
+    }
+}
+```
+{% /multicode %}
+
+## Decrement a field {% #decrement-field %}
+
+Use the `decrementDocumentAttribute` method to decrease numeric fields:
+
+{% multicode %}
+```client-web
+const result = await databases.decrementDocumentAttribute(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    '<DOCUMENT_ID>',
+    'credits', // attribute
+    5 // value
+);
+
+console.log(result);
+```
+```client-flutter
+final document = await databases.decrementDocumentAttribute(
+    databaseId: '<DATABASE_ID>',
+    collectionId: '<COLLECTION_ID>',
+    documentId: '<DOCUMENT_ID>',
+    attribute: 'credits',
+    value: 5
+);
+```
+```client-apple
+let document = try await databases.decrementDocumentAttribute(
+    databaseId: "<DATABASE_ID>",
+    collectionId: "<COLLECTION_ID>",
+    documentId: "<DOCUMENT_ID>",
+    attribute: "credits",
+    value: 5
+)
+```
+```client-android-kotlin
+val document = databases.decrementDocumentAttribute(
+    databaseId = "<DATABASE_ID>",
+    collectionId = "<COLLECTION_ID>",
+    documentId = "<DOCUMENT_ID>",
+    attribute = "credits",
+    value = 5
+)
+```
+```server-nodejs
+const result = await databases.decrementDocumentAttribute(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    '<DOCUMENT_ID>',
+    'credits', // attribute
+    5 // value
+);
+```
+```server-python
+result = databases.decrement_document_attribute(
+    database_id = '<DATABASE_ID>',
+    collection_id = '<COLLECTION_ID>',
+    document_id = '<DOCUMENT_ID>',
+    attribute = 'credits', # attribute
+    value = 5 # value
+)
+```
+```graphql
+mutation {
+    databasesDecrementDocumentAttribute(
+        databaseId: "<DATABASE_ID>",
+        collectionId: "<COLLECTION_ID>",
+        documentId: "<DOCUMENT_ID>",
+        attribute: "credits",
+        value: 5
+    ) {
+        _id
+        _collectionId
+        _databaseId
+        _createdAt
+        _updatedAt
+        _permissions
+        data
+    }
+}
+```
+{% /multicode %}
+
+# Set constraints and bounds {% #set-constraints-and-bounds %}
+
+You can set minimum and maximum bounds for numeric fields to prevent invalid values. Use the optional `min` and `max` parameters:
+
+{% info title="Collection configuration required" %}
+Constraints must be configured at the collection level when defining attributes.
```

Comment:
This is incorrect, while you can configure constraints for the attribute in general, the increment/decrement routes allow to specify the value per request. Needs to be done that way since we don't fetch the document first, so we don't know yet at validation time if we're going to exceed the min/max